airbyte-integrations/connectors/source-open-exchange-rates/integration_tests/sample_config.json

```
@@ -0,0 +1,5 @@
+{
+  "start_date": "2022-11-13",
+  "base": "USD",
+  "access_key": "Your API Access Key"
```

In your spec.yaml, you defined this key as `app_id`

```suggestion
  "app_id": "Your API Access Key"
```